components/Pagination/__tests__/Pagination.test.js

```diff
@@ -8,29 +8,39 @@ describe('Pagination', () => {
   afterEach(cleanup);
 
   it('should render with required props', () => {
-    createSnapshotTest(<Pagination currentPage={3} totalPages={5} />);
+    createSnapshotTest(<Pagination currentPage={3} totalPages={5} maxElements={11} />);
   });
 
-  it('should not have seperatorStart if currentPage - 1 is less than 6', () => {
-    const { queryByTestId } = render(<Pagination currentPage={6} totalPages={11} />);
+  it('should not have seperatorStart if currentPage - 1 is less than (maxElements + 1) / 2', () => {
```

Comment:
Any way you can make these test names a bit less mathy / bit more quickly skimmable?

Reply:
Yeah sure, also as we are going to drop `maxElements` the test names would be simplified.